bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/handler/BaseModuleHandler.java

```diff
@@ -41,4 +43,32 @@ public void setCallback(ModuleHandlerCallback callback) {
     public void dispose() {
         this.callback = null;
     }
+
+    /**
+     * Returns the {@link Module}, which belongs to the handler.
+     *
+     * @return {@link Module}, which belongs to the handler
+     */
+    public T getModule() {
```

Comment:
This becomes public API, so I do not understand why that is necessary for fixing an NPE?
It also does not seem to be used at all.

Reply:
Yes, you are right. this method is not needed at all. I removed it.